tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -130,3 +130,18 @@ stages:
         RunOnnxRuntimeTests: ${{ parameters.RunOnnxRuntimeTests }}
         # Liqun is fixing them
         RunStaticCodeAnalysis: true
+
+- stage: training_x64_debug_on_device_training
+  dependsOn: []
+  jobs:
+    - template: templates/win-cpu-ci.yml
+      parameters:
+        BuildConfig: 'Debug'
+        EnvSetupScript: setup_env.bat
+        buildArch: x64
+        additionalBuildFlags: --enable_training --enable_training_on_device --build_wheel
```

Comment:
do we need build_wheel here?

Reply:
I don't think we need. I kept it to be consistent with the other stages in this file.
Can remove.